### PR TITLE
Refresh feature info after editing feature

### PR DESCRIPTION
**Pull request by geertplaisier**
Solution for  #1115

### CODE REVIEW

File: viewer/src/main/webapp/viewer-html/components/ExtendedFeatureInfo.js [hunk 1]
```diff
@@ -73,6 +73,7 @@ Ext.define ("viewer.components.ExtendedFeatureInfo",{
         this.getViewerController().mapComponent.getMap().addListener(viewer.viewercontroller.controller.Event.ON_GET_FEATURE_INFO,this.onFeatureInfoStart,this);
         //listen to a extent change
         this.getViewerController().mapComponent.getMap().addListener(viewer.viewercontroller.controller.Event.ON_CHANGE_EXTENT, this.onChangeExtent,this);
+        this.config.viewerController.addListener(viewer.viewercontroller.controller.Event.ON_EDIT_SUCCESS, this.featureEdited, this);
```
Comment (mprins) [inline viewer/src/main/webapp/viewer-html/components/ExtendedFeatureInfo.js]: odd to use `this.config.viewerController` here while bove we use `this.getViewerController()`

File: viewer/src/main/webapp/viewer-html/components/FeatureInfo.js [hunk 2]
```diff
@@ -66,7 +67,8 @@ Ext.define ("viewer.components.FeatureInfo",{
         this.getViewerController().mapComponent.getMap().addListener(viewer.viewercontroller.controller.Event.ON_GET_FEATURE_INFO,this.onFeatureInfoStart,this);
         //listen to a extent change
         this.getViewerController().mapComponent.getMap().addListener(viewer.viewercontroller.controller.Event.ON_CHANGE_EXTENT, this.onChangeExtent,this);
-        return this;        
+        this.config.viewerController.addListener(viewer.viewercontroller.controller.Event.ON_EDIT_SUCCESS, this.featureEdited, this);
```
Comment (mprins) [inline viewer/src/main/webapp/viewer-html/components/FeatureInfo.js]: odd to use `this.config.viewerController` here while bove we use `this.getViewerController()`

**Review by mprins — COMMENTED** [review]
does this also work when a feature is deleted? (need to check)

**Comment by mprins** [comment]
~~needs backport to 5.2.x branch~~ seems to be part of v5.2.0-rc2

**Comment by mprins** [comment]
I've rebased to current master and force-pushed this

**Comment by mprins** [comment]
I tested this and it works for attribute edits, but fails to update the extended feature panel  when a feature is deleted.

When a feature is deleted the panel should be erased and the marker should be removed from the map

**Review by mprins — CHANGES_REQUESTED** [review]
it should also work when features are deleted, see other comments

**Review by mtoonen — APPROVED** [review]

**Review by mprins — APPROVED** [review]

**Pull request merged**